### PR TITLE
Add manage of highlight that render as chp backgroundColor

### PR DESCRIPTION
**Pull request by msturari**
Add parsing of \highlight parameter that is rendered as chp backgroundColor

### CODE REVIEW

**Comment by zoehneto** [comment]
Thank you for your contribution. 
Please add a test document showing the feature in action (see https://github.com/tbluemel/rtf.js#tests for how to generate a test case). You also have to run `npm run build` and commit the new version of the js bundles so the build passes (unfortunately travis ci doesn't post the build results in the PR anymore, see https://travis-ci.org/github/tbluemel/rtf.js/builds/763900576).

**Comment by msturari** [comment]
Hi, I have added bundle files and a test document.

File: test/rtf-test-files/colors/expected-metadata.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (zoehneto) [inline test/rtf-test-files/colors/expected-metadata.json:1]: This doesn't match the meta data in the rtf file, please regenerate the test case

File: test/rtf-test-files/colors/expected.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<div style="min-height: 8pt; text-align: left;">
```
Comment (zoehneto) [inline test/rtf-test-files/colors/expected.html:1]: I assume your editor formatted this file. That breaks the test though. When you regenerate the test case this should be fixed automatically.

**Review by zoehneto — CHANGES_REQUESTED** [review]
For some reason the generated source maps don't match what's generated on travis ci (also not what is generated when I run `npm run build` locally). You may want to try reinstalling the dependencies with `npm ci` so you get the correct versions.

**Review by zoehneto — APPROVED** [review]
The source maps were still incorrect, I've regenerated them and pushed that change. Will merge now.

**Pull request merged**